### PR TITLE
Dockerized test client

### PR DESCRIPTION
**Pull request by SebastianSchildt**
Dockerfile and helper script to containerize the vss-testclient.

We can later build it in CI (and push to dockerhub), so it is easier to get started playing with kuksa.val

### CODE REVIEW

**Review by wenwenchenbosch — APPROVED** [review]

File: clients/vss-testclient/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+FROM python:3.8-alpine AS build-env
+
+
+RUN mkdir /app
+RUN mkdir /app/vssclient
+COPY vss-testclient/testclient.py /app/vssclient/
+ADD vss-testclient/tokens /app/vssclient/tokens
+ADD common /app/common
+WORKDIR /app/vssclient
+RUN pip install --target=./ --no-cache-dir websockets cmd2 pygments
```
Comment (wenwenchenbosch) [inline clients/vss-testclient/Dockerfile:22]: use Pipfile?

File: clients/vss-testclient/README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ pipenv run python testclient.py
 ### Without Pipenv
 Ensure all the dependencies are installed.
 ```sh
-pip3 install websockets cmd2
+pip3 install websockets cmd2 pygments
```
[the remark this answers is not in the document]
Comment (SebastianSchildt) [reply clients/vss-testclient/README.md]: I added a requirements.txt

File: clients/vss-testclient/Dockerfile (hunk 1 above)
Comment (SebastianSchildt) [reply clients/vss-testclient/Dockerfile]: Using the requiremnts.txt now

**Comment by SebastianSchildt** [comment]
Added a requirements.txt. That is still somehow duplicated with the pipfile, but still useful for people who do not have pipenv installed. Also used it in Dockerfile

**Comment by SebastianSchildt** [comment]
leave this at just a "Self-service" dockerfile, more magic ways to build(x) via Jenkinsfile come in a separate commit

File: clients/vss-testclient/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+cmd2~=1.3
+websockets~=8.1
+pygments~=2.7
```
Comment (wenwenchenbosch) [inline clients/vss-testclient/requirements.txt:3]: CQ for these libraries of the specified version are still missing?

Comment (SebastianSchildt) [reply clients/vss-testclient/requirements.txt:3]: Submitted.

**Pull request merged**